### PR TITLE
Genome wide scores prf95 2

### PR DESCRIPTION
**Pull request by at7**
Suggestions for adding CADD and GERP scores from file. At the moment scores are only returned by variation feature.

### CODE REVIEW

File: modules/Bio/EnsEMBL/Variation/DBSQL/AnnotationFileAdaptor.pm [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+  ## explicit use
+
+  # get Annotation File Adaptor
+  my $annotation_file_adaptor = $reg->get_adaptor('human', 'variation', 'AnnotationFile');
+  my $va  = $reg->get_adaptor('human', 'variation', 'variation');
+  my $v = $va->fetch_by_name('rs699');
+  my $vf = $v->get_all_VariationFeatures->[0];
+
+  my $gerp_file = $annotation_file_adaptor->fetch_by_type('gerp');
```
Comment (ens-lgil) [inline modules/Bio/EnsEMBL/Variation/DBSQL/AnnotationFileAdaptor.pm]: Should be replaced by `fetch_by_annotation_type` in the description of this file and few others

File: modules/Bio/EnsEMBL/Variation/DBSQL/AnnotationFileAdaptor.pm [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+    die "Found more than one annotation file for annotation type $annotation_type for species $species and assembly $assembly\n";
+  } else {
+    die "Could not find annotation file for annotation type $annotation_type for species $species and assembly $assembly\n";
+  }
+}
+
+sub _get_gerp_annotation_file {
+  my $self = shift;
+  my $hash = shift;
+  return Bio::EnsEMBL::Variation::GERPFile->new(
```
Comment (ens-lgil) [inline modules/Bio/EnsEMBL/Variation/DBSQL/AnnotationFileAdaptor.pm]: Method  `_get_gerp_annotation_file` quite similar to `_get_cadd_annotation_file`.
Maybe we could use a more generic method using `GERPFile` or `CADDFile` depending on the type in the parameter

File: modules/Bio/EnsEMBL/Variation/AnnotationFile.pm [hunk 3]
```diff
@@ -0,0 +1,175 @@
+=head1 LICENSE
```
Comment (ens-lgil) [inline modules/Bio/EnsEMBL/Variation/AnnotationFile.pm]: As this module is not used on its own (serve as base for `GERPFile` and `CADDFile`, maybe we could rename it `BaseAnnotationFile`

**Comment by at7** [comment]
Thanks for the feedback. This will not go out in release/95 anymore. We will discuss the code in the meeting on 26th. Please bring any ideas you have.

**Comment by at7** [comment]
Updated in #271

**Pull request closed without merging**